src/dot-components/Wishlist/wishlist.css

```diff
@@ -0,0 +1,53 @@
+@import url("https://techtonic-ui.netlify.app/styles.css");
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,300;0,400;0,500;0,900;1,300&display=swap');
+
+body{
+    font-family: "Montserrat", sans-serif;
+}
```

Comment:
you can add this style in root.css itself to avoid repetition

Reply:
Okay.